sdcm/tester.py

```diff
@@ -2961,16 +2961,17 @@ def clean_resources(self):
 
         self.destroy_credentials()
 
-    @silence(name='Save node schema')
     def save_nodes_schema(self):
         if self.db_cluster is None:
             self.log.info("No nodes found in the Scylla cluster")
 
         self.log.info("Save nodes user schema in the files")
         for node in self.db_cluster.nodes:
```

Comment:
Need also to do it in parallel on all the nodes.

JFYI, 60 DB nodes setup: https://jenkins.scylladb.com/job/scylla-staging/job/valerii/job/vp-longevity-aws-custom-d2-workload1-multidc-big/17/consoleFull

Reply:
we might consider also to limit it, and not take from all of the nodes
maybe just the first 10 ?

Reply:
Why do even need to take it from all nodes?
The schema is the same on all nodes.

We can just pick one node and run it there.

Reply:
I can do it just for one live node

Reply:
@juliayakovlev also after talking a bit with roy, let put the file inside the SCT bundle, and not the node.
so you don't need to send it back to the node.